Automattic/wp-calypso
# Remove the OlarkChat component

#### Proposed Changes

We're no longer using Olark for pre-sales or other flows.
For JP cloud ref, see #49882
For other uses, see pau2Xa-40L-p2#comment-12053

Removing it includes:
* Remove the implementation on the logged-out theme showcase (it was removed from signup in #67378)
* Remove the implementation on the JP Cloud (it was disabled there in #49882)

#### Testing Instructions

The Olark chat isn't currently showing up, because it's not staffed or not configured. 
But the following URLs are affected by this change (check them for errors/issues): 
- `/themes/` while logged out
- https://cloud.jetpack.com/dashboard / http://jetpack.cloud.localhost:3001/partner-portal
- https://cloud.jetpack.com/partner-portal / http://jetpack.cloud.localhost:3001/partner-portal

#### Pre-merge Checklist




Related to #
